src/components/EditorContent/ImagePreview.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useRef, useState } from "react";
+
+import { useOnClickOutside } from "neetocommons/react-utils";
+import { CloseCircle } from "neetoicons";
+import { Button, Spinner } from "neetoui";
+
+const ImagePreview = ({ imagePreviewUrl, setImagePreviewUrl }) => {
+  const [isLoading, setIsLoading] = useState(true);
+
+  const imagePreviewRef = useRef(null);
+
+  useOnClickOutside(imagePreviewRef, () => setImagePreviewUrl(null), {
+    enabled: true,
+  });
+
+  return (
+    <div className="image-preview-wrapper">
+      <div className="close-button">
+        <Button
+          icon={CloseCircle}
```

Comment:
Use the `Close` icon from neeto-icons.

```suggestion
          icon={Close}
```